src/sgame/sg_bot_util.cpp

```diff
@@ -35,6 +35,8 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 #include <glm/gtx/vector_angle.hpp>
 
 static Cvar::Range<Cvar::Cvar<int>> g_bot_defaultSkill( "g_bot_defaultSkill", "Default skill value bots will have when added", Cvar::NONE, 5, 1, 9 );
+static Cvar::Cvar<int> g_bot_alienAimModifier = Cvar::Cvar<int>( "g_bot_alienAimModifier", "make bots of alien team slower to aim", Cvar::NONE, 250 );
```

Comment:
I suggest naming it `g_bot_alienAimDelay`. With "modifier" it's unclear whether a higher number means better or worse aiming. Description could be like "alien bots' aiming time in milliseconds"

Reply:
done